buildifier/buildifier.go

```diff
@@ -24,11 +24,17 @@ import (
 	"io/ioutil"
 	"os"
 	"runtime"
-	"sort"
 	"strings"
+	"sync"
 
-	build "github.com/bazelbuild/buildifier/core"
-	"github.com/bazelbuild/buildifier/differ"
+	core "github.com/bazelbuild/buildifier/core"
```

Comment:
On Tue, Jul 19, 2016 at 12:44 PM, Fabrizio Milo <notifications@github.com>
wrote:

> In buildifier/buildifier.go
> <https://github.com/bazelbuild/buildifier/pull/6#discussion_r71376887>:
>
> >
> > -	build "github.com/bazelbuild/buildifier/core"
> > -	"github.com/bazelbuild/buildifier/differ"
> > +	core "github.com/bazelbuild/buildifier/core"
>
> When the package has a different name from the folder, is a bit misleading
> from the go perspective. So I was thinking to rename the entire package to
> core or rename the folder to build. Either of the two solutions are fine
> to me.
>
The reason I chose core and not build is the OSX problem of having a file
'BUILD' and a directory 'build'.
But the reason I aliased the name has to do with the way the code is
imported at Google, so as I wrote, please just leave this alone as

build "github.com/bazelbuild/buildifier/core"




> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/bazelbuild/buildifier/pull/6/files/fff89537f62dd2868d478a4b9d512b23826994ec..6e9312af29eb5a6745707967feb7f04032d9d27b#r71376887>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALF-0mzjyIol6taVjc6Ir9o54Dzliwa1ks5qXP8EgaJpZM4JNFI0>
> .
>